lib/virt_autotest/common.pm

```diff
@@ -220,6 +220,15 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             location => 'http://mirror.suse.cz/install/SLP/SLE-15-SP6-Full-GM/x86_64/DVD1/',
             boot_firmware => 'efi',    # EFI version of SLES 15 SP6
         },
+        sles15sp6efiseves => {
```

Comment:
The name is a little long... Will it help to add sth like _ or - to separate the words?